modules/gapi/src/streaming/onevpl/utils.cpp

```diff
@@ -9,6 +9,9 @@
 #include <algorithm>
 #include <sstream>
 
+#include <opencv2/gapi/streaming/onevpl/utils.hpp>
+#include <opencv2/gapi/util/throw.hpp>
```

Comment:
how does it work without these included?

Reply:
Fixing it

Reply:
fixed